WcaOnRails/app/assets/stylesheets/teams-and-committees.scss

```diff
@@ -0,0 +1,36 @@
+.badge.team-member-badge {
+  margin-right: 1em;
+  margin-bottom: 1em;
+  color: #555;
+  background-color: #dddddd;
+}
+
+.team-member-name {
+  padding-left: 1em;
+  display: inline-block;
+}
+
+.badge.team-leader-badge {
+  background-color: #007bff;
+  color: white;
+  padding-top: 6px;
+  vertical-align: top;
+  span.team-leader-subtext {
+    font-style: oblique;
+    margin-top: 6px;
+    display: block;
+  }
+}
+
+.badge.officer-badge {
+  @extend .badge, .team-leader-badge;
+  background-color: black;
+  color: white;
+}
+
+.badge.team-email {
+  background-color: #e64503;
+  a {
+    color: white;
+  }
+}
```

Comment:
Seems like there's no common property for .badge alone. Is there a reason to group them?

Reply:
Badge is a [Bootstrap component](https://getbootstrap.com/docs/4.3/components/badge/) which is why they all have the badge class. The grouping is just for organizational purposes, but this can be changed if you think it's less clean.

Reply:
That's fine.